src/services/armService.ts

```diff
@@ -231,7 +231,7 @@ export class ArmService extends BaseService {
 
       // This is a json path expression
       // Learn more @ https://goessner.net/articles/JsonPath/index.html#e2
-      const appSettingsPath = "$.resources[?(@.kind==\"functionapp\")].properties.siteConfig.appSettings";
+      const appSettingsPath = "$.resources[?(@.type==\"Microsoft.Web/sites\")].properties.siteConfig.appSettings";
```

Comment:
Why is this changing?

Reply:
This is a `jsonpath` query, to find the function app settings. The `kind` property was being used in the search, but it changed, so this search query is no longer valid. Now using the `type` property since that is still valid and unchanging.